source/atlas-cli-deploy-fts.txt

```diff
@@ -426,9 +426,26 @@ Search <avs-types-vector-search>`.
          .. code-block:: sh
 
             atlas deployments search indexes create --file indexDef-vector.json
+            
+         You must specify the path to the file if the
+         ``indexDef-vector.json`` isn't in the directory from where you
+         wish to run the command.
 
       #. Specify the deployment and press :kbd:`Enter`..
```

Comment:
Small observation: Is this supposed to have two periods? I see this across the page.

Reply:
good catch. Fixed.